### DISCUSSION TITLE
Figure out correct font face

### DISCUSSION THREAD

**Discussion by iamvoidcoder** [zed-industries/zed#6601] [Feature Requests]
In absence of font picker, how does one work out the correct font face name to feed to buffer_font_family setting?

Some fonts are trivial but some are a total disaster. E.g. JatBrains Mono Nerd Regular. The actual font file name is **JetBrainsMonoNerdFontMono-Regular.ttf**. When installed on a Mac it is seen as **JetBrainsMono Nerd Font Mono** family in Mac Font Book viewer app, system font pickers and elsewhere. But whatever I try for buffer_font_family setting it is not picking it up.

Any simple rules to work out the correct font name string for buffer_font_family setting?

Thanks

I do hope eventually a proper GUI font picker will appear in Zed. Even the simplest of simplest basic editors like TextEdit and Notepad allow to visually pick the font.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
